### DISCUSSION TITLE
A strange issue with Gmail

### DISCUSSION THREAD

**Discussion by yellow1912** [maizzle/framework#659] [Q&A] [answered]
Gmail is acting weird, I wonder if I'm doing something wrong:

My span element has these classes:

`font-bold text-base w-6/12 text-right`

Which is converted by maizzle into:
`width: 50%; text-align: right; font-size: 1rem; font-weight: 700; --tw-text-opacity: 1; color: rgb(51 51 51 / var(--tw-text-opacity));`

This style, when sent to Gmail, is completely removed. I think it's caused by the `--tw-text-opacity: 1; color: rgb(51 51 51 / var(--tw-text-opacity));` but I'm unsure.

**Comment by cossssmin** [comment, marked as answer]
Gmail doesn't support that CSS and, when Gmail encounters unsupported/invalid/"invalid" CSS it completely removes it, yes.

Make sure you have text opacity turned off, the Starter does that by default:

https://github.com/maizzle/maizzle/blob/master/tailwind.config.js#L141

**Reply by yellow1912** [reply]
Thank you, got it. Fixed by properly disabling those core plugins.
